src/dump_xyz.cpp

```diff
@@ -159,8 +176,22 @@ void DumpXYZ::write_header(bigint n)
   if (me == 0) {
     if (!fp) error->one(FLERR, "Must not use 'run pre no' after creating a new dump");
 
-    auto header = fmt::format("{}\n Atoms. Timestep: {}", n, update->ntimestep);
-    if (time_flag) header += fmt::format(" Time: {:.6f}", compute_time());
+    std::string header;
+
+    if (extxyz_flag) {
+      header = fmt::format("{}\nTimestep={}", n, update->ntimestep);
+      if (time_flag) header += fmt::format(" Time={:.6f}", compute_time());
+      header += fmt::format(" pbc=\"{} {} {}\"", domain->xperiodic ? "T" : "F",
```

Comment:
Shouldn't there be more fields here for extxyz, e.g. potential energy etc.?
You'd have to create the necessary compute instances internally similar to how e.g. the Thermo class does it.